src/bintray.coffee

```diff
@@ -1,7 +1,10 @@
 _ = require "lodash"
 Rest = require "./rest"
 common = require "./common"
+sendFile = require("restler").file
```

Comment:
do you still need restler?

Reply:
Restler? Yes. This `sendFile` helper? No. I'll remove it from the file

Reply:
It's kind of annoying to remove it, commit and generate a new hash for Android, so it'll only be updated in our master branch.